### DISCUSSION TITLE
Problems with genservext.py

### DISCUSSION THREAD

**Discussion by cuncil1** [jgyates/genmon#1038] [Questions]
### Expected Behavior

The genservext.py should serve to send you to a webpage in the static folder

### Actual Behavior

It is not working and provides the following text:

Not Found
The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

### Steps to Reproduce (including precondition)

I copied the sample text with the "testit" function and a webpage that was in the static folder. I can access the webpage if I write it in the address. The webpage is called gencontrol.html and the IP for my device is 192.168.68.81, therefore if I add 192.168.68.81/gencontrol.html it works, but if I tried to do it with testit, it does not.

### Your Environment

- Generator Model: Generac Sync 2
- Genmon Logs: Submit via the About page
- Genmon Version: V1.19.01

**Comment by jgyates** [comment]
I don't have a lot to go on to provide you any feedback. Please show the exact contents of the file on the pi. what is the output of this command:

     cd genmon
     cat genservext.py

Once you make a change to genservext.py, did you restart genmon? 
Is the file gencontrol.html referenced in your gensevext.py file? The example uses a file named myhtml.html but you never mentioned modifying the example.
Is the file gencontrol.html located in ~./genmon/static folder?
Are the permissions for gencontrol.html the same as the other files in the ~./genmon/static folder
What is the output of /var/log/genserv.py
What is the exact command you are using in your browser?

**Reply by jgyates** [reply]
I am going to close this issue but feel free to post to this thread with status or additional questions.
